general/administration/hardware-acceleration.md

```diff
@@ -49,6 +49,117 @@ The hardware acceleration is available immediately for media playback. No server
 
 Each hardware acceleration type, as well as each Jellyfin installation type, requires different setup options before it can be used. It is always best to consult the FFMpeg documentation on the acceleration type you choose for the latest information.
 
+##Install & Setup for Debain/Ubuntu Nvidia NVENC
+First off we need to figure out what the most up to date driver is for your NVIDIA Card is.
+So for this we need to go to the below link.
+
+[NVIDIA Driver Update](https://www.nvidia.com/Download/index.aspx)
```

Comment:
This you can also use the driver from the repo `nvidia-driver-440` for example

Reply:
i realize this but with the repo its not always either up to date or issues could arise from them as seen with the 970 i just worked through on today on jellyfin chat. He had what he though was the most up to date but it wasnt

Reply:
Well it works just fine for my 960 so no problem there, it should work for 90% of the people